src/PrettyPrinter.php

```diff
@@ -52,7 +53,35 @@ public function endTest(Test $test, float $time): void
         }
```

Comment:
The preceding four lines are no longer necessary

Reply:
Thats true but i wont make any changes at the moment on this until one of the maintainers has given feedback on this in general. But thanks, i would also change some other things after looking at this code after such a time :D